### ISSUE TITLE
module tty missing

### ISSUE THREAD

**Issue by abhayastudios** [telerik/backend-services-js-sdk#7]
I am trying to use everlive-sdk 1.6.12 but when I try to run in iOS simulator I get this:

```
file:///app/tns_modules/debug/./node.js:6:18: JS ERROR Error: Could not find module 'tty'. Computed path '/Users/abhaya/Library/Developer/CoreSimulator/Devices/044E7BDF-67CE-4F38-97E7-B5F73E1794DE/data/Containers/Bundle/Application/4D46429A-7040-4D24-994A-07BA2B5D85B4/ns-test.app/app/tns_modules/tty'.
```

Any advice?

**Comment by AntonDobrev** [comment]
@abhayastudios 

Indeed, this error appears in the latest NativeScript environment. We are aware of this issue and I believe it will be addressed in a newer version of the SDK.

Meanwhile you can install the SDK locally by downloading the SDK package as explained [here](http://docs.telerik.com/platform/backend-services/javascript/nativescript)

**Comment by AntonDobrev** [comment]
This is fixed in the just released 1.7.1. You can import the npm package and require the SDK like:

```
var Everlive = require('everlive-sdk');
...

var el = new Everlive( { appid: 'myappId', scheme: 'https' });
```
